server/channels/app/server.go

```diff
@@ -208,7 +208,6 @@ func NewServer(options ...Option) (*Server, error) {
 	// Depends on step 1 (s.Platform must be non-nil)
 	s.initEnterprise()
 
-	// Needed to run before loading license.
```

Comment:
Do we know the context around this comment?

Reply:
@isacikgoz might know. I think this was added when we shifted things around to create the platform service.

Reply:
I think this is related to this incident: https://mattermost.atlassian.net/browse/MM-42378 But I think after we move the session cache to the Platform, we should be in a safe state.